### PR TITLE
Create test project

### PR DESCRIPTION
**Pull request by Brett208**
Didn't make changes to makefile.

Should look through TestModule and move some tests over to unit tests as appropriate. I don't think we will be able to remove TestModule if we want to produce a DLL for testing op2ext's external hooks, but open to the idea if it exists.

### CODE REVIEW

File: IpDropDown.h [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#pragma once
-
```
Comment (Brett208) [inline IpDropDown.h]: Looks like this file did not pickup on the rename. Not sure it matters being 3 lines long.

File: TestModule/CompatibilityTest.c [hunk 2]
```diff
@@ -1,4 +1,4 @@
 // Tests if the external inteface has basic compatibility with C compiler.
```
Comment (Brett208) [inline TestModule/CompatibilityTest.c]: File rename not recognized on GitHub.

**Comment by Brett208** [comment]
Is it still worth reviewing source code files so they contain an empty line at the end of each file. It doesn't do anything for me except keep GitHub from putting red Xs at the bottom of source code files. I understand some Unix based tools might require it? If there is a chance someone would want to use these tools on this project, I'm okay continuing, just want to make sure I'm not wasting time tracking them down and committing them.

-Brett

File: TestModule/CompatibilityTest.c [hunk 3]
```diff
@@ -1,4 +1,4 @@
 // Tests if the external inteface has basic compatibility with C compiler. 
 // IE compiles without errors and warnings.
 
-#include "op2ext.h"
\ No newline at end of file
+#include "include/op2ext.h"
```
Comment (DanRStevens) [inline TestModule/CompatibilityTest.c]: Having the `include/` folder explicitly in the `#include` is probably not ideal. I think usually an include folder setting would be added to the project.

Comment (Brett208) [reply TestModule/CompatibilityTest.c]: I'm happy either way. 

-Brett

**Pull request closed without merging**